[2024-04-20T16:07:56.972+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-20T16:07:57.063+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2024-04-13T00:00:00+00:00 [queued]>
[2024-04-20T16:07:57.137+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2024-04-13T00:00:00+00:00 [queued]>
[2024-04-20T16:07:57.149+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-20T16:07:57.378+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): third_task> on 2024-04-13 00:00:00+00:00
[2024-04-20T16:07:57.391+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2363) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-20T16:07:57.402+0000] {standard_task_runner.py:63} INFO - Started process 2380 to run task
[2024-04-20T16:07:57.397+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'third_task', 'scheduled__2024-04-13T00:00:00+00:00', '--job-id', '297', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmphx15r8m8']
[2024-04-20T16:07:57.435+0000] {standard_task_runner.py:91} INFO - Job 297: Subtask third_task
[2024-04-20T16:07:57.647+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v3.third_task scheduled__2024-04-13T00:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-20T16:07:57.922+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-13T00:00:00+00:00'
[2024-04-20T16:07:57.931+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-20T16:07:58.002+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-20T16:07:58.005+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task3 and will be running after task 1']
[2024-04-20T16:07:58.113+0000] {subprocess.py:86} INFO - Output:
[2024-04-20T16:07:58.115+0000] {subprocess.py:93} INFO - hey, I am task3 and will be running after task 1
[2024-04-20T16:07:58.116+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-20T16:07:58.116+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-20T16:07:58.175+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=third_task, execution_date=20240413T000000, start_date=20240420T160757, end_date=20240420T160758
[2024-04-20T16:07:58.233+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-20T16:07:58.374+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-20T16:07:58.417+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
